ledger-service/http-json/src/it/scala/http/HttpServiceWithPostgresIntTest.scala

```diff
@@ -26,136 +20,6 @@ class HttpServiceWithPostgresIntTest
 
   override def wsConfig: Option[WebsocketConfig] = None
 
-  implicit lazy val logHandler: log.LogHandler = dao.logHandler
```

Comment:
so the issue was that we ran all startup tests only for postgres?

Reply:
Actually I still don’t get it. The regular tests should still create the schema and then startup. Do we skip schema validation in that case? That seems like something we might want to address. The startup shouldn’t work differently in the tests than it works outside of the tests.

Reply:
Yes, the startup specific tests were only run for postgres. However that they were not run for Oracle was more or less me not completely knowing & understanding the test suite yet.

Now that we are talking about it, you're right. This was never changed, however I will change this and hope nothing breaks.

Reply:
The reason why the tests were still passing is simply because the code did not contain an error but just an semantic bug. So it would always reinitialize the cache (if allowed by the startup mode).

Reply:
Ok addressed your request with a484f89. Let's see whether all tests still pass with that, locally I only ran HttpServiceWithPostgresIntTest which passed :)

Reply:
> So it would always reinitialize the cache (if allowed by the startup mode).
meaning that with `createSchema=false` which all of our existing customers are using (as it was the only option so far), it would never start?

Reply:
Yes

Reply:
Side note, it would start if one uses `CreateAndStart`.